src/tests/ftest/util/soak_utils.py

```diff
@@ -427,8 +427,8 @@ def launch_vmd_identify_check(self, name, results, args):
         for value in list(result['response']['host_storage_map'].values()):
             if value['storage']['smd_info']['devices']:
                 for device in value['storage']['smd_info']['devices']:
-                    if device['led_state'] != "QUICK_BLINK":
-                        failing_vmd.append([device['tr_addr'], value['hosts']])
+                    if device['ctrlr']['led_state'] != "QUICK_BLINK":
+                        failing_vmd.append([device['ctrlr']['pci_addr'], value['hosts']])
```

Comment:
@mjean308 FYI on this one. Not sure if soak_smoke would cover this

Reply:
That is only executed by the `test_soak_online_harassers` and that particular harasser - `vmd-identify-check` - is commented out currently.